apps/ledger-live-desktop/src/renderer/components/OperationsList/AmountCell.jsx

```diff
@@ -33,7 +33,10 @@ class AmountCell extends PureComponent<Props> {
   render() {
     // eslint-disable-next-line no-unused-vars
     const { currency, unit, operation } = this.props;
-    const amount = getOperationAmountNumber(operation);
+    const amount =
+      currency.id === "cosmos"
```

Comment:
This isn't the right approach as we don't want to add a "if cosmos" in this generic part of the code.
The fix for his ticket is most likely on the live-common side, somewhere in the cosmos family implementation.